histomics_stream/configure.py

```diff
@@ -322,7 +322,8 @@ def __call__(self, slide):
                     "in Zarr storage."
                 )
 
-        slide["level"] = level
+        int_level = int(level + 0.5)
+        slide["level"] = int_level if abs(level - int_level) < 1e-6 else level
```

Comment:
For what it is worth, I think we use 1e-4 as the threshold in geojs.

Reply:
I just force-pushed with `1e-4` as the threshold.  I also put a `math.floor` call in there because it appears that Python's `int` cast rounds towards zero rather than rounding towards negative infinity.

Reply:
I probably would have used `int(round(value))`, but that also moves values that have exactly .5 as the fraction toward zero.

Reply:
We reject `int_level` if it is farther than 1e-4 from the returned `level`, so what happens with 0.5 won't affect us here.  I've gone with `int(round(level))` with this latest force push.